exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/MathFunctions.java

```diff
@@ -33,22 +33,40 @@
 import org.apache.drill.exec.expr.holders.VarCharHolder;
 
 public class MathFunctions {
-
+/*
```

Comment:
Why is this commented out and why does it replace the existing Float8 implementation?